src/cmd-cloud-prune

```diff
@@ -136,14 +145,35 @@ def main():
                 match action:
                     case "cloud-uploads":
                         prune_cloud_uploads(current_build, cloud_config, args.dry_run)
-                    case "build":
-                        raise NotImplementedError
-                        # print(f"Deleting key {prefix}{build.id} from bucket {bucket}")
-                        # Delete the build's directory in S3
-                        # S3().delete_object(args.bucket, f"{args.prefix}{str(current_build.id)}")
+                    # Prune through images that are not mentioned in images-keep
                     case "images":
-                        raise NotImplementedError
-            build.setdefault("policy-cleanup", []).append("cloud-uploads")
+                        images_to_keep = policy.get(stream, {}).get("images-keep", [])
```

Comment:
we pull these out of the policy quite a few times.. Maybe we should just do this once and then use the variable throughout.

Reply:
fixed now